index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <div class="body-collection">
+      <header>
+        <div class="sign-in">
+          <div class="sign-in__already">
+            <div>Already have an Intuit account?</div>
+            <div><a>I forgot my User ID or Password</a></div>
+          </div>
+          <div class="sign-in__button">
+            <a class="sign-in-button"
+              ><img class="lock-img"
+                src="https://img.icons8.com/material-outlined/24/000000/lock.png"
+              />Sign in</a
+            >
+          </div>
+        </div>
+      </header>
+      <main>
+        <section>
+          <div class="form-title">
+            <a class="logo-pic" href="#"></a>
+            <div class="logoes">
+              <div class="mint-logo"></div>
+              <div class="mint-verb"></div>
+              <div class="quickbooks-logo"></div>
+              <div class="quickbooks-verb"></div>
+              <div class="turbotax-logo"></div>
+              <div class="turbotax-verb"></div>
+            </div>
+
+            <h1 class="title">Create an Intuit account</h1>
+          </div>
+          <form action="#" enctype="multipart/form-data">
+            <div class="form">
+              <fieldset>
+                <div class="input-form">
+                  <label for="email">Email address</label
+                  ><input type="email" name="email" required />
```

Comment:
The label `for` attribute must contain the input `id` value. For ecample:

`<label for="the-value">Foo</label>`
`<input id="the-value">`

Kindly adjust for all other `label` and `input` elements. You can check [this article on w3schools](https://www.w3schools.com/tags/att_label_for.asp) to see how it was formatted.